Formula/shairport-sync.rb

```diff
@@ -1,9 +1,10 @@
 class ShairportSync < Formula
   desc "AirTunes emulator. Shairport Sync adds multi-room capability."
   homepage "https://github.com/mikebrady/shairport-sync"
-  url "https://github.com/mikebrady/shairport-sync/archive/2.8.3.6.tar.gz"
-  sha256 "f59c3bfc2b95af60b822a9729abd94f33330ff81090b00194cf0c999c59133b6"
-  head "https://github.com/mikebrady/shairport-sync.git"
+  url "https://github.com/mikebrady/shairport-sync/archive/2.8.4.tar.gz"
+  sha256 "80dd94c5f37b43e9b157dd5335f8edaf11109859d0144e0046c7b86fe88f6547"
+  head "https://github.com/mikebrady/shairport-sync.git",
+    :branch => "development"
```

Comment:
>We've already started adopting 4 spaces for this sort of thing so it lines up with the opening "

ew.

> I loathe :branch falling onto its own line

Not sure I follow the logic there. I find it much easier to miss the non-default branch when it's all the way on the right hand side, even possibly passed the eightieth column (:scream:) than when it's indented on the following line, where it's harder not to notice.